.github/workflows/build.yml

```diff
@@ -11,7 +11,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: '14.x'
+          node-version: '14'
+          check-latest: true
```

Comment:
This is slightly less performant, it forces the job to download the latest node version each time. 
The links to other related issues for playwright and node version are in the commit message.